src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/NotificationsType.php

```diff
@@ -26,27 +26,35 @@
 
 namespace PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\Administration;
 
+use PrestaShopBundle\Form\Admin\Type\MultistoreConfigurationType;
 use PrestaShopBundle\Form\Admin\Type\SwitchType;
 use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class NotificationsType extends TranslatorAwareType
 {
+    public const FIELD_SHOW_NOTIFS_NEW_ORDERS = 'show_notifs_new_orders';
+    public const FIELD_SHOW_NOTIFS_NEW_CUSTOMERS = 'show_notifs_new_customers';
+    public const FIELD_SHOW_NOTIFS_NEW_MESSAGES = 'show_notifs_new_messages';
+
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('show_notifs_new_orders', SwitchType::class, [
+            ->add(self::FIELD_SHOW_NOTIFS_NEW_ORDERS, SwitchType::class, [
                 'label' => $this->trans('Show notifications for new orders', 'Admin.Advparameters.Feature'),
```

Comment:
I think it has logical wording issue?
Shouldn't it be `of new orders` ?

Reply:
Hello @zuk3975, the current string looks fine to me.